src/main/groovy/com/palantir/gradle/docker/DockerSetupTask.groovy

```diff
@@ -30,32 +28,14 @@ class DockerSetupTask extends DefaultTask {
 
     @Override
     public String getDescription() {
-        return 'Verify that Docker-related environment variables are set'
+        return 'Verify that Docker daemon is running.'
     }
 
     @TaskAction
     public void action() {
-        if (!OperatingSystem.current().isLinux()) {
-            def dockerTlsVerify = System.getenv('DOCKER_TLS_VERIFY')
-            def dockerHost = System.getenv('DOCKER_HOST')
-            def dockerCertPath = System.getenv('DOCKER_CERT_PATH')
-            def dockerMachineName = System.getenv('DOCKER_MACHINE_NAME')
-            def error = ''
-            if (!System.env.DOCKER_TLS_VERIFY) {
-                error += 'DOCKER_TLS_VERIFY not set.\n'
-            }
-            if (!System.env.DOCKER_HOST) {
-                error += 'DOCKER_HOST not set.\n'
-            }
-            if (!System.env.DOCKER_CERT_PATH) {
-                error += 'DOCKER_CERT_PATH not set.\n'
-            }
-            if (!System.env.DOCKER_MACHINE_NAME) {
-                error += 'DOCKER_MACHINE_NAME not set.\n'
-            }
-            if (error != '') {
-                throw new GradleException(error += 'Please make sure your Docker VM is running.')
-            }
+        project.exec {
```

Comment:
You get what I pasted in the description and some version information: 
```
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```

Can change to docker ps, the output will be less noisy. Or in general get rid of this task, all the other ones will fail with the same message when cannot connect.